src/main/java/core/basesyntax/model/ma/Coach.java

```diff
@@ -1,5 +1,10 @@
 package core.basesyntax.model.ma;
 
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "coaches")
 public class Coach extends Person {
     public enum Track {
```

Comment:
better to put @Enumerated(EnumType.STRING) annotation as well on the field where you use it